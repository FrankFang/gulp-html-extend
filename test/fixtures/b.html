<!-- @@master=a.html-->
<!-- @@block=content-->
<main>
    my content

    <!-- @@placeholder=main -->
</main>
<!-- @@close-->
<!-- @@block=footer-->
<footer>
    my footer
</footer>
<!-- @@close-->
