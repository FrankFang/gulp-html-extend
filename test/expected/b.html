<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
        <title></title>
    </head>
    <body>
<!-- start content -->
<main>
    my content

    <!-- @@placeholder=main -->
</main>

<!-- end content -->
<!-- start footer -->
<footer>
    my footer
</footer>

<!-- end footer -->
    </body>
</html>
